Allow operators to have zero wires (#7327)
PennyLaneAI/pennylane: doc/releases/changelog-dev.md

* Alias for Identity (`I`) is now accessible from `qml.ops`.
  [(#7200)](https://github.com/PennyLaneAI/pennylane/pull/7200)

> * PennyLane no longer validates that operations that can exist on any number of wires
>   have at least one wire.

Reviewer: ```suggestion
* PennyLane no longer requires operations that can act on any number of wires
  to have at least one wire.
```

Author: ```suggestion
* PennyLane no longer validates that an operation has at least one wire.
```

* Alias for Identity (`I`) is now accessible from `qml.ops`.
  [(#7200)](https://github.com/PennyLaneAI/pennylane/pull/7200)

* PennyLane no longer validates that an operation has at least one wire.

>   [(#7327)](https://github.com/PennyLaneAI/pennylane/pull/7327)

Author: Because it was bad software design?

> * PennyLane no longer validates that an operation has at least one wire.
>   [(#7327)](https://github.com/PennyLaneAI/pennylane/pull/7327)

Author: ```suggestion
* PennyLane no longer validates that an operation has at least one wire, as having this check required the abstract
  interface to maintain a list of special implementations.
  [(#7327)](https://github.com/PennyLaneAI/pennylane/pull/7327)
```

>   [(#7327)](https://github.com/PennyLaneAI/pennylane/pull/7327)

Reviewer 2: Yeah that's nice! And that "list of special implementations" is not really needed?
